nextflow_schema.json

```
@@ -96,20 +110,28 @@
                     "fa_icon": "fas g",
                     "description": "Remove PolyG sequences (length of 10 or more)",
                     "type": "boolean"
-                }
-            }
-        },
-        "adapterqc_options": {
-            "title": "Adapter QC Options",
-            "type": "object",
-            "properties": {
+                },
                 "adapterqc_mismatches": {
                     "fa_icon": "fas not-equal",
                     "description": "The number of mismatches allowed (in percentage) [default: 0.1; 0.0<=x<=0.9]",
                     "type": "number",
                     "default": 0.1,
                     "minimum": 0.0,
                     "maximum": 0.9
+                },
+                "save_qc_passed_reads": {
+                    "fa_icon": "fas fa-save",
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Save intermediate qc read files containing all reads that passed the filters.",
+                    "help": "By default, filtered read FastQ files after QC will not be saved to the results directory. Specify this flag (or set to true in your config file) to copy these files to the results directory when complete."
+                },
+                "save_qc_failed_reads": {
+                    "fa_icon": "fas fa-save",
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Save intermediate qc read files containing all reads that failed the filters.",
```

```suggestion
                    "description": "Save intermediate QC read files containing all reads that failed the filters.",
```
same for the following